### ISSUE TITLE
No tests

### ISSUE THREAD

**Issue by bergie** [noflo/noflo-webaudio#31]
@automata we have bunch of PRs open for this repo, but I'm hesitant to merge any given the zero tests this project has

**Comment by automata** [comment]
@bergie yeah, this lib is really WIP. Would be nice to update it and write proper tests. We also need to decide if that's the right way to go for WebAudio support, or if we should use alternatives like [noflo-flocking](https://github.com/automata/noflo-flocking) or [noflo-gibber](https://github.com/automata/noflo-gibber).

**Comment by bergie** [comment]
We're using this on Flowhub examples, so in that sense would be great to modernize and put on NPM
